app/javascript/components/import/Import.js

```diff
@@ -116,16 +143,22 @@ class Import extends React.Component {
                   aria-valuemax={this.state.phased.length}></div>
               </div>
             )}
-            <h5>Please review the monitorees that are about to be imported below. You can individually accept each monitoree, or accept all at once.</h5>
+            <h5>
+              Please review the monitoree records that are about to be imported. You can individually accept or reject each record below. You can choose to
+              import all unique records or all records (including duplicates) by selecting Accept.
+            </h5>
             <Button
               variant="primary"
               className="btn-lg my-2"
               onClick={() =>
                 this.handleConfirm(
-                  'This will not import already rejected or re-import already accepted monitorees listed below. If duplicates are detected, you will be presented with an option to include them.'
+                  `This will import all records listed below that you did not manually accept or reject.If potential duplicates have been detected, check the box if you would like to import them.`
                 )
               }>
-              Accept All
+              Accept
+            </Button>
+            <Button variant="primary" className="btn-lg my-2 ml-2" onClick={() => this.stopImport()}>
```

Comment:
So `onClick={() => this.stopImport()}` can just be `onClick={this.stopImport}`. You don't need an arrow function expression if you've already defined a function that does what you want without arguments.

Reply:
Tried doing this, but React kept throwing `Cannot update during an existing state transition (such as within `render`). Render methods should be a pure function of props and state.
    in Import`

Not 100% sure why, but just going to leave them as `() => `